File: 283-move-zeroes/move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
       int i=0;
       int n=nums.length;
       int c=0;
       for(int j=0;j<n;j++){
       if(nums[j]!=0)nums[i++]=nums[j];
       } 
       while(i<n) nums[i++]=0;
    }
}

File: 283-move-zeroes/move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        i=0
        c=0
        for j in nums:
            if j!=0:
                nums[i]=j
                i+=1
            
        for j in range(i,len(nums)):
            nums[i]=0
            i+=1
        